### PR TITLE
Tara i2

### PR DESCRIPTION
**Pull request by tarads443**

### CODE REVIEW

File: functions/common.py [hunk 1]
```diff
@@ -24,3 +25,40 @@ def mod(dividend: float, divisor: float) -> float:
     while dividend < divisor:
         dividend += divisor
     return dividend
+
+
```
Comment (SwatiPareek) [inline functions/common.py:29]: I created a file called display_output in the PR for our rad->deg conv function. 
I was thinking maybe all of these types of functions should be in that file. 
What do you think?

**Comment by EmanuelUnity3D** [comment]
I guess the built-in conversions weren't allowed for this project? Everything looks good to me. It just needs to be refactored to meet PEP-8 standards. Also, do we include the tests now or later?

File: functions/common.py (hunk 1 above)
Comment (tarads443) [reply functions/common.py:29]: Are you referring to the factorial() function or the decimal_to_binary_integer() and decimal_to_binary_decimal() functions?

**Comment by tarads443** [comment]
Hey so I changed the code to conform to PEP-8 and I added unit tests. 

Two things I'll add: 
- I'll move the functions over to output_display()
- Need to figure out how to add unit tests for negative numbers. 

I'll do this in my second push because I want to push this PR as it's a bit old and I'll create a new one for the two items above. 

Do you mind seeing if you are okay to approve?

File: functions/common.py [hunk 2]
```diff
@@ -28,6 +29,60 @@ def factorial(argument: int) -> int:
...
 
 
+def mod(dividend: float, divisor: float) -> float:
+    # TODO not sure on the typing for this,
+    # and should check if can be optimized
+    while dividend > divisor:
+        dividend -= divisor
+    while dividend < divisor:
+        dividend += divisor
+    return dividend
```
Comment (Emancus09) [inline functions/common.py:39]: I would just use the built-in mod because it's a built-in operator. Unless Python's mod works differently from this one?

File: functions/common.py [hunk 3]
```diff
@@ -27,8 +27,8 @@ def mod(dividend: float, divisor: float) -> float:
     return dividend
 
 
-# Function to convert a whole number to binary
 def decimal_to_binary_integer(value):
+    """Returns a string that contains the binary result of an integer"""
```
Comment (Emancus09) [inline functions/common.py:43]: The docstring convention that we adopted documents the parameters, return values, and possible exceptions thrown in addition to the description of the function itself. I'm not really partial to one convention over another but I would add those things just for the sake of consistency.

File: tests/test_common.py [hunk 4]
```diff
@@ -20,6 +20,9 @@ def test_factorial_float_raise_error(self):
         with self.assertRaises(the_exception.InputError):
             common.factorial(56.3)
 
+    def test_factorial_float_raise_error(self):  # 0b is removed from bin result for all unit tests
+        self.assertEqual((common.convert_to_binary(6)), bin(6).replace("0b", ""))  # Test for positive integer
+        self.assertEqual((common.convert_to_binary(-6)), bin(-6).replace("0b", ""))  # Test for negative integer
```
Comment (Emancus09) [inline tests/test_common.py:25]: How come we aren't testing floating point values?

File: functions/common.py (hunk 3 above)
Comment (tarads443) [reply functions/common.py:43]: Thanks I'll change this. Can I change this in the next PR as it's a minor change?

File: tests/test_common.py (hunk 4 above)
Comment (tarads443) [reply tests/test_common.py:25]: This is something that I need to add but was going to do it in the next PR as I have to figure out how because the bin() function doesn't accept floating point values.

Comment (EmanuelUnity3D) [reply tests/test_common.py:25]: Ah ok

File: functions/common.py (hunk 3 above)
Comment (EmanuelUnity3D) [reply functions/common.py:43]: Sure as long as we remember it.
